thirdeye-ui/package.json

```diff
@@ -147,6 +148,8 @@
         "postcss": "8.3.6",
         "prettier": "2.4.0",
         "prettier-plugin-organize-imports": "2.3.3",
+        "react-refresh": "0.14.0",
+        "react-refresh-typescript": "2.0.9",
```

Comment:
I think you only need typescript one

Reply:
![Screenshot 2024-01-12 at 16 11 46](https://github.com/startreedata/thirdeye/assets/20799363/a6354e21-3ae0-4234-8d34-cece25df8cfd)

Without `react-refresh`, the project doesn't even start